components/suggestion/emoticon_provider.jsx

```diff
@@ -109,9 +109,10 @@ export default class EmoticonProvider extends Provider {
     findAndSuggestEmojis(text, partialName, resultsCallback) {
         const recentMatched = [];
         const matched = [];
-
-        const emojiMap = getEmojiMap(store.getState());
-        const recentEmojis = getRecentEmojis(store.getState());
+        const state = store.getState();
+        const skintone = state.entities?.preferences?.myPreferences['emoji--emoji_skintone']?.value || 'default';
```

Comment:
It might be easier to use the `get` selector from `selectors/entities/preferences` here. If we're using this in multiple places, we could also make a new selector that just wraps `get('emoji', 'emoji_skintone', 'default')` as well. 0/5 though.

Reply:
i'll keep it in mind and see if I can change it in any of the next PRs, thanks for pointing it out